src/app/site/site.component.html

```diff
@@ -162,7 +162,7 @@ <h2 *ngIf="selectedDelivery">
     </div>
     <nextime-delivery-date
       *ngIf="selectedOrderDate && siteId && selectedAddress && selectedOrder"
-      [orderDate]="selectedOrderDate | json"
+      [orderDate]="selectedOrderDate"

```

Comment:
@vzanza what is the reason for removing this?

Reply:
the JsonPipe transforms it into a json format string which breaks the date validation - do you see any problem removing it?